edi/models/edi_exchange_consumer_mixin.py

```diff
@@ -139,6 +142,17 @@ def edi_create_exchange_record(self, exchange_type_id):
 
     def _has_exchange_record(self, exchange_type, backend=False, extra_domain=False):
         """This function is useful when generating the configuration"""
```

Comment:
can you change the docstrings as you are here? :wink:

Reply:
Done